src/mempools.c

```diff
@@ -365,9 +365,10 @@ struct memblock_s {
     } u;
 };
 
-#define MEMBLOCK_LIMIT (128)
```

Comment:
smalloc.c has a different size limit for small blocks (68 bytes). It would be a bit nicer to get this constant from the allocator. But we can also check for the reason why there a two different limits...